docs/fast_data/faq/debezium.md

```diff
@@ -0,0 +1,133 @@
+
+# Debezium Oracle connector
+
+This section addresses questions specific to the Oracle connector.
+
+## How to add the Oracle JDBC to the Debezium Docker image?
+Due to licensing reasons, if you want to use Debezium to connect to an Oracle database, you need to build your own image and manually include the Oracle JDBC. Use the following Dockerfile (ensure the JDBC group, version, and MD5 are correct):
+```Dockerfile
+FROM quay.io/debezium/connect:2.5
+
+ENV KAFKA_CONNECT_JDBC_DIR=$KAFKA_CONNECT_PLUGINS_DIR/kafka-connect-jdbc
+
+# These should point to the driver version to be used
+ENV MAVEN_DEP_DESTINATION=$KAFKA_HOME/libs \
+    ORACLE_JDBC_REPO="com/oracle/database/jdbc" \
+    ORACLE_JDBC_GROUP="ojdbc10" \
+    ORACLE_JDBC_VERSION="19.21.0.0" \
+    ORACLE_JDBC_MD5=55544d916412e364112695485d078a66
+
+# Download Oracle JDBC driver
+RUN docker-maven-download central "$ORACLE_JDBC_REPO" "$ORACLE_JDBC_GROUP" "$ORACLE_JDBC_VERSION" "$ORACLE_JDBC_MD5"
+
+USER kafka
+```
+
+## Oracle connector fails with ORA-00600 error
+
+If your Debezium Oracle connector fails with the `ORA-00600` error, it indicates an internal Oracle database error. Although it is challenging to pinpoint the cause, restarting the connector typically resolves the issue. To automate recovery from this error, you can set up a cron job to:
+
+- Check the status of the Oracle connector using the `GET/connectors/:connectorId/status` API.
+- If the connector's task is in `FAILED` status, restart the connector and its tasks using the `POST/connectors/:connectorId/restart?includeTasks=true` API.
+
+## Oracle connector fails with error "Online REDO LOG files or archive log files do not contain the offset scn. Please perform a new snapshot."
+This error usually occurs in two scenarios:
+
+1. Your Oracle connector stays offline longer than the retention period of the Oracle REDO and archive logs. When you restart the connector, it tries to find the SCN where it last stopped, and if it is no longer available, you get this error.
+2. The initial snapshot takes longer than the Oracle log retention period, leading to this error right after the snapshot ends, before starting the mining session.  
+
+Since this error necessitates a new initial snapshot, reducing its likelihood is crucial:
+
+1. Implement a cron job as suggested earlier to avoid prolonged offline periods for the connector.
+2. Measure the initial snapshot duration and request your DBA to increase the log retention period to cover this duration.
+
+## Oracle connector fails with error "ORA-01950: no privileges on tablespace 'USERS'"
```

Comment:
also here, it looks more a troubleshooting rather than a faq

Reply:
see previous comment